src/nxt_time_parse.c

```diff
@@ -317,8 +317,7 @@ nxt_term_parse(const u_char *p, size_t len, nxt_bool_t seconds)
     enum {
         st_first_digit = 0,
         st_digit,
-        st_letter,
-        st_space,
+        st_next,
```

Comment:
In string parsing, enums are usually used to represent different character types, `st_space` represents spaces.
And they are placed in the order they are parsed. Take `st_space` as an example, it means it's expected after the digit is parsed.
If `next` is used in `switch` and we used `goto next`, it makes sense to use `next` but not `space` if `next` is used.
Based on them, I feel it's good to keep it.